src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteModel.js

```diff
@@ -156,7 +157,7 @@ export default class StandardRouteModel extends BaseModel {
          */
         this._entryCollection = null;
 
-        return this._init(standardRoute);
```

Comment:
I was tracking down a bug,  Accidentally left it out after it was found. Adding it back in. I Should have commented it out and added the `this._init(standardRoute);` as its own line with a comment showing that though